Maintainers_Guide.md

```diff
@@ -0,0 +1,62 @@
+
+## Responsibilities
+
+The primary responsible of a maintainer is ensuring the infrastructure that the BIDS community uses is handled and kept up to date.
+Here we list several examples of common responsibilities for a BIDS maintainer.
+Please note that one maintainer need not do all of these tasks.
+They may be spread across the team according to the initiative of each maintainer and/or the direction of the lead maintainer.
+
+* Triaging issues and PRs then notifying appropriate reviewers in a timely manner
+* Facilitating discussions
+* Ensuring the Contributors list is up to date and assisting contributors with adding their credits
+     * Perform community inquiries to ensure contributors are credited in the [Contributors appendix](https://bids-specification.readthedocs.io/en/stable/99-appendices/01-contributors.html)
+     * Verify contributing members (e.g., author of a pull request) are also identified in the [Contributors appendix](https://bids-specification.readthedocs.io/en/stable/99-appendices/01-contributors.html)
+     * Ultimately, what constitutes a contribution is left to the discretion of the maintainers
+* Assisting with voting and elections
+* Proposing and implementing enhancements to the infrastructure
+
+Maintainers service time will be agreed upon between the lead maintainer and the interested maintainer.
+Maintainers will be reviewed every 3 - 6 months.
+This will be determined on a case by case basis at the discretion of the lead maintainer.
+There is no limit to the number of appointments a member may obtain.
+
+## Admission process
+
+### Eligibility
+
+To become eligible to join the Maintainers group the prospective member would have been contributing to BIDS for at least 3 months and have a good overview of the content and infrastructure surrounding BIDS.
+Contributing to BIDS can be in the form of submitting pull requests, commenting on specification or project documents, or actively participating on a BIDS extension proposal (BEP).
+
+### Joining the team
+
+Interested and eligible prospective members and the Maintainers Group lead will discuss a potential appointmentship and clarify any concerns either one may have.
+If an agreement can be formed, the prospective maintainer’s application will be submitted to the BIDS Steering Group for review.
+Upon a successful review, the Contributors group will vote to finalize the appointment.
+A successful vote will have at least 20% of the Contributors voted with the result based on a majority decision.
+Only the initial appointment needs a vote.
+
+### Appointment review period
+
+After a member’s appointment period has been completed, the Maintainers Group lead and member will discuss the member’s status and evaluate granting another appointment.
+The appointment time period will be agreed on between the lead and member.
+
+### Lead Maintainer
+
+The Maintainers Group will have a self-elected lead.
```

Comment:
will it be up to the lead maintainer to introduce a new successor? (i.e., to step down?)

... or do we want to have a max time for lead maintainers?

I am in favor of the former option (lead maintainer steps down when he/she wants and has a successor)

Reply:
I wonder if rather than appointing a successor, all current maintainers would be able to vote on the next lead maintainer ? The new lead would then be elected by simple majority vote, with the past lead able to vote twice in the case of a tie. I've seen this done in several committees and it seems to work.

Reply:
That seems like a good balance to me.

Reply:
sounds good - but this election would be initiated by the lead maintainer once he/she wants to step down?

Reply:
I would think so.